src/plots/polar/polar.js

```diff
@@ -993,11 +993,13 @@ proto.updateRadialDrag = function(fullLayout, polarLayout) {
             var moduleCalcDataVisible = Lib.filterVisible(moduleCalcData);
             var _module = moduleCalcData[0][0].trace._module;
             var polarLayoutNow = gd._fullLayout[_this.id];
+            var isGL = Registry.traceIs(k, 'gl');
+
+            if(isGL && _this._scene) _this._scene.clear();
```

Comment:
is it possible to not have `_this._scene` at this point if there's a `gl` trace?

BTW this would be clearer with `traceType` instead of `k`.

I don't see a problem at the moment because there's only one `gl` type that can be here, but for future reference seems like if we ever add another we'll have to make sure only the first one clears the scene.

Reply:
Thanks for tip. Improvements in -> b81885d7315b1d16e609e7d1ca7af3311ea3dbaa